config/projects.yml

```diff
@@ -214,11 +214,10 @@ doctrine-mongodb-admin-bundle:
 doctrine-orm-admin-bundle:
   branches:
     master:
-      php: ['7.2', '7.3', '7.4']
+      php: ['7.3', '7.4']
       target_php: '7.3'
       versions:
         symfony: ['4.4']
-        sonata_admin: ['3']
```

Comment:
maybe it is better to use 4 here

Reply:
I don't know how this config works, but the version 4 doesn't exist.

Reply:
```
- php: '7.3'
      env: SONATA_ADMIN=4.*
```
will fail

Reply:
This works by adding an extra build for 4.x-dev of sonata admin. I guess now it is not required , but when you release 4, it is better to also test with unreleased 4.x (to ensure the latest Pr does not break compatibility)

Reply:
Yes, but it does 2 things:
```
{% for package_name,package_versions in versions %}
{% for version in package_versions %}
    - php: '{{ target_php|default(php|last) }}'
      env: {{ package_name|upper }}={{ version }}.*
{% endfor %}
{% if package_versions|length > 0 %}
    - php: '{{ target_php|default(php|last) }}'
      env: {{ package_name|upper }}='dev-master as {{ package_versions|last }}.x-dev'
{% endif %}
{% endfor %}
```
Will add 
```
- php: '7.3'
   env: SONATA_ADMIN=4.*
```

And 
```
{% for package_name,package_versions in versions %}
{% if package_versions|length > 0 %}
    - env: {{ package_name|upper }}='dev-master as {{ package_versions|last }}.x-dev'
{% endif %}
{% endfor %}
```
Will add an extra build for 4.x-dev.

If we want the second, the first will be added and will obviously fail.
So we need to wait for the 4.x release in order to re-add this.